tests/e2e_logging_pbft.py

```diff
@@ -34,8 +34,16 @@ def run(args):
 
             LOG.debug("Write/Read on primary")
             with primary.user_client(format="json") as c:
-                check_commit(c.rpc("LOG_record", {"id": 42, "msg": msg}), result=True)
-                check_commit(c.rpc("LOG_record", {"id": 43, "msg": msg2}), result=True)
+                check_commit(
+                    c.rpc("LOG_record", {"id": 42, "msg": msg}),
+                    result=True,
+                    force_sign=True,
```

Comment:
AIUI, we shouldn't need this. Instead, as @olgavrou mentioned, we should pass the value of `--sig-max-ms` and `--sig-max-tx` (passed to `cchost`) to the PBFT. When this is done, `check_commit()` should eventually succeed (as long as we keep passing `1000` as `--sig-max-ms`, just like for Raft).